clients/java/signalr/src/main/java/DefaultJsonProtocolHandShakeMessage.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+public class DefaultJsonProtocolHandShakeMessage {
+    String protocol = "json";
```

Comment:
Is the default visibility `public`? I think we should be explicit here even if that's the default.